code/controllers/subsystem/input.dm

```diff
@@ -1,15 +1,28 @@
-SUBSYSTEM_DEF(input)
+VERB_MANAGER_SUBSYSTEM_DEF(input)
 	name = "Input"
-	wait = 1 //SS_TICKER means this runs every tick
 	init_order = INIT_ORDER_INPUT
 	init_stage = INITSTAGE_EARLY
 	flags = SS_TICKER
 	priority = FIRE_PRIORITY_INPUT
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
 
+	use_default_stats = FALSE
+
 	var/list/macro_set
 
-/datum/controller/subsystem/input/Initialize()
+	///running average of how many clicks inputted by a player the server processes every second. used for the subsystem stat entry
+	var/clicks_per_second = 0
+	///count of how many clicks onto atoms have elapsed before being cleared by fire(). used to average with clicks_per_second.
+	var/current_clicks = 0
+	///acts like clicks_per_second but only counts the clicks actually processed by SSinput itself while clicks_per_second counts all clicks
+	var/delayed_clicks_per_second = 0
+	///running average of how many movement iterations from player input the server processes every second. used for the subsystem stat entry
+	var/movements_per_second = 0
```

Comment:
not sure I agree with this as a stat, seems potentially misleading

Reply:
whats misleading about it?

Reply:
https://github.com/tgstation/tgstation/pull/65589#discussion_r916417710 mostly. players often complete inputs, what's interesting is the cost of moving itself, rather then the cost of holding down the button, no?